models/experimental/functional_stable_diffusion/tt2/ttnn_functional_cross_attention_down_block_2d_new_conv.py

```diff
@@ -0,0 +1,123 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
```

Comment:
It's very tough to see the changes this way (when adding new files). Do you think you could add the files in their original format as one PR first, and then have this one modify them? Then we can discuss differences

Reply:
I've split it into two commits. The first commit creates the new files, but with the exact same content as the original ones. The second commit modifies them.

Just the modifications are visible [here](https://github.com/tenstorrent/tt-metal/pull/8786/commits/e3619b7361fef75ea5c4f007cd00a624956b18d5).

Reply:
I left comments on the second commit.

Reply:
Now that we're going back and forth on changes, it's again getting really tough to review. Can you please create two PRs? The first one should just be the new files, unchanged from the original ones, and in the second one you can rebase with all the new changes.